python-code/set-sample/sample.py

```diff
@@ -1,3 +1,5 @@
 VALUE1 = 2.0
 import os
 import math
```

Comment:
This is a Unusedimportmath
```suggestion 
```

 from our GitHub Enhacement Bot